### DISCUSSION TITLE
Texteditor cursor issue as show in image

### DISCUSSION THREAD

**Discussion by heejit** [cogentcore/core#1230] [General]
left side cursor 
![image](https://github.com/user-attachments/assets/b5df044d-62a7-402d-9b0f-8db5c33035aa)
![image](https://github.com/user-attachments/assets/c496036e-232b-4d05-86f2-f9ed5a83c6e3)

**Comment by kkoreilly** [comment]
Thank you for reporting this. We will work on fixing this.
